### DISCUSSION TITLE
Streaming support

### DISCUSSION THREAD

**Discussion by vasudevanubrolu** [ggml-org/whisper.cpp#2156] [Q&A]
Hi 

I am looking for a stable version to support streaming use cases. The current model requires a longer 1 to 30 secs audio chunks and does the stream the results. Hence it requires VAD to split the streaming audio and retrieve transcripts. In the recent GPT 4o launch they updated to shorter audio chunk but I am still not sure how we can use whisper for streaming use cases in production use cases.

**Comment by ErcinDedeoglu** [comment]
I'm experiencing a similar issue with stream.exe on Windows, as there is no satisfactory build documentation. Could someone please tell me if there are any ready-to-download binaries available?

**Comment by rbunny87** [comment]
@ErcinDedeoglu did you tried stream command ?

**Comment by torivaha** [comment]
The binary "stream" is deprecated, and whisper-stream is not in the built/bin-folder. How may I build / aquire this binary?

**Reply by luiznpi** [reply]
where is the great whisper-stream code ? the stream example does not merge/update the last wrong transcription...
